seurat-find-transfer-anchor.R

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env Rscript
+
+suppressPackageStartupMessages(require(optparse))
+suppressPackageStartupMessages(require(workflowscriptscommon))
+
+#set parsing options
+
+option_list = list(
+ 
+  
+   make_option(
+    c("--query-file"),
```

Comment:
Could you provide short-form parameters as well? Like for reference-file below there's a '-i' short form.